src/VisualizationTests/HelixWatch3DViewModelTests.cs

```diff
@@ -1189,7 +1189,7 @@ private Watch3DView FindFirstWatch3DNodeView()
             return views.Last();
         }
 
-        [Test]
+        [Test, Category("Failure")]
```

Comment:
Why is this failing now?

Reply:
The two meshes do not exists any more within the `Element3DDictionary`.   Previously this RenderPackageCache would have added `var_88f09e6c057f4a5c95a7f4f0b25161e2:0:0:0":mesh` and `var_88f09e6c057f4a5c95a7f4f0b25161e2:0:0:1:mesh`.  Now as far as the Helix view is concerned there is only one mesh object related to this node identifier in the `Element3DDictionary`.  That is `var_88f09e6c057f4a5c95a7f4f0b25161e2:mesh` I think.  The indexes are no longer added here as this new implementation only adds one bucket for mesh, curve, and vertex data to the Helix dictionary per node.  In fact that is why this is so much faster.  1 point bucket vs potentially 100K is much much faster.

Reply:
And much more memory efficient.

Reply:
Why not remove this test in that case?

Reply:
That is good.  I just marked them as fail initially as to clearly communicate behavioral changes